examples/layer-extent.js

```diff
@@ -43,6 +43,8 @@ var map = new ol.Map({
   })
 });
 
+map.addInteraction(new ol.interaction.DragRotateAndZoom());
```

Comment:
As @tschaub mentioned, alt-shift-drag always allows you to rotate. So this could be removed.